CSP [style|style]-src-[attr|elem] example fixes (#22277)
mdn/content: files/en-us/web/http/headers/content-security-policy/style-src-elem/index.md

      <th scope="row">{{CSP("default-src")}} fallback</th>
      <td>
        <p>
          Yes.

>           If this directive is absent, the user agent will look for the {{CSP("style-src")}} directive, and if both of them are absent, fallback to <code>default-src</code> directive.

Reviewer: ```suggestion
          If this directive is absent, the user agent will look for the {{CSP("style-src")}} directive, and if both of them are absent, fall back to <code>default-src</code> directive.
```

## Examples

### Violation cases

Given this CSP header:

```http
Content-Security-Policy: style-src-elem https://example.com/
```

> the following stylesheets are blocked and won't load:

Reviewer: ```suggestion
…the following stylesheets are blocked and won't load:
```

```html
<link href="https://not-example.com/styles/main.css" rel="stylesheet" />

<style>
  #inline-style {
    background: red;
  }
</style>

<style>
  @import url("https://not-example.com/styles/print.css") print;
</style>
```

> as well as styles loaded using the {{HTTPHeader("Link")}} header:

Reviewer: ```suggestion
…as well as styles loaded using the {{HTTPHeader("Link")}} header:
```
